src/main/java/uk/gov/pay/directdebit/payers/model/AccountNumber.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.pay.directdebit.payers.model;
+
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+/**
+ * A UK bank account number with no dashes or spaces e.g "12345678"
+ * Guaranteed to be well-formed (six to eight Arabic numerals
+ * inclusive) but not necessarily valid (in the sense of representing
+ * a real bank account)
+ */
+public class AccountNumber {
+
+    private static final Pattern SIX_TO_EIGHT_ARABIC_NUMERALS = Pattern.compile("[0-9]{6,8}");
```

Comment:
I am happy to immediately have the information that they are not roman numerals. Good job

Reply:
You have to be careful! In some languages, `\d` in a regex matches more than just the Arabic numerals 0–9: https://blog.mariusschulz.com/2015/10/21/matching-digits-in-net-regex